src/search/sr_optimizers.c

```
@@ -203,11 +247,30 @@ void sr_optimizer_config_apply(const sr_optimizer_config *cfg)
   if (cfg->max_iters > 0) {
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)